BuildResidentialHPXML/measure.rb

```diff
@@ -1382,6 +1420,74 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setUnits('W')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument.makeChoiceArgument('heat_pump_perf_data_heating_cap_type', perf_data_capacity_type_choices, false)
+    arg.setDisplayName('Heat Pump: Heating Detailed Performance Data Capacity Type')
+    arg.setDescription('Capacity type of heat pump heating detailed performance data.')
+    arg.setUnits('Capacity')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('heat_pump_perf_data_heating_outdoor_temps', false)
+    arg.setDisplayName('Heat Pump: Heating Detailed Performance Data Outdoor Temperatures')
+    arg.setDescription('Outdoor temperatures of heat pump heating detailed performance data. If multiple performance data points, use a comma-separated list.')
+    arg.setUnits('deg-F')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('heat_pump_perf_data_heating_min_caps', false)
+    arg.setDisplayName('Heat Pump: Heating Detailed Performance Data Minimum Capacities')
+    arg.setDescription('Minimum capacities of heat pump heating detailed performance data. If multiple performance data points, use a comma-separated list.')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('heat_pump_perf_data_heating_max_caps', false)
+    arg.setDisplayName('Heat Pump: Heating Detailed Performance Data Maximum Capacities')
+    arg.setDescription('Maximum capacities of heat pump heating detailed performance data. If multiple performance data points, use a comma-separated list.')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('heat_pump_perf_data_heating_min_cops', false)
+    arg.setDisplayName('Heat Pump: Heating Detailed Performance Data Minimum Efficiency COP Values')
+    arg.setDescription('Minimum efficiency COP of heat pump heating detailed performance data. If multiple performance data points, use a comma-separated list.')
+    arg.setUnits('W/W')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('heat_pump_perf_data_heating_max_cops', false)
+    arg.setDisplayName('Heat Pump: Heating Performance Data Maximum Efficiency COP Values')
+    arg.setDescription('Maximum efficiency COP of heat pump heating detailed performance data. If multiple performance data points, use a comma-separated list.')
+    arg.setUnits('W/W')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeChoiceArgument('heat_pump_perf_data_cooling_cap_type', perf_data_capacity_type_choices, false)
```

Comment:
No need for separate heating and cooling inputs, no one is going to want to use capacities for one and fractions for the other.